docs/source/gateway.md

```diff
@@ -90,6 +90,19 @@ The Fabric Gateway `Evaluate` and `Endorse` methods make gRPC requests to peers
 
 Each Gateway SDK also provides a mechanism for setting timeouts for each gateway method when invoked from the client application.
 
+## Gateway Service Routing with Kubernetes 
+
+In typical Fabric deployments on Kubernetes, each peer node is exposed via a single Kubernetes `Service` instance and resolved using a Kube DNS.  This technique is sufficient for the Fabric Gateway and application clients to resolve individual peers within a Fabric network.  In cases requiring high-availability and/or client connection load-balancing, Kubernetes can be configured with an additional `Service` resource bound to multiple gateway peers.  Using an HA topology, gateway clients may reference a set of peer nodes with a single DNS alias.
+
```

Comment:
an observation: is it worth pushing the gateway client strongly here?  a) make them well known. but also b) this approach works due to the way gateway has a single point of (network) contact with Fabric.  Really not sure this would well with previous SDKs?